depot-tools-git/depot-tools-git.install

```diff
@@ -11,5 +11,12 @@ post_install() {
 	Then re-login as your current user or log your terminal in to the newly created group:
 
 	$ newgrp depot_tools
+
+	Zsh completions
+	source /usr/share/zsh/plugins/zsh-goodies/_gclient in .zshrc
+
+	Bash completions
+	source /home/lichen/Downloads/pkg/depot-tools-git/usr/share/bash-completion/completions/* in .bashrc
+
```

Comment:
This whole section can be dropped, the ZSH completions placed in the right place will work out of the box, as will the bash ones. Also this path has your personal user directory as the path...not something that goes in packaging.

Reply:
Will do.